swift_browser_ui_frontend/src/components/CFooter.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="c-footer">
+    <nav class="navbar">
+    <div class="navbar-menu is-active">
+        <div class="navbar-start" >
+          <div class="col-3">
+            <div class="row navbar-item " >
+              <span class="largetext">
+              {{ $t("message.footerMenu.groupandprogram") }}
+            </span>
+            </div>
+            <div class="row navbar-item smalltext">
+              <a class="linktext"
+              href="https://csc.fi"
+              :alt="$t('message.footerMenu.csc')"
+            >{{
+              $t("message.footerMenu.csc")
+            }}</a>
+            </div>
+          </div>        
+        </div>
+    
+        <div class="navbar-end" id="rightmenu"> 
+            <div class="navbar-item smalltext">
+                <a class="linktext"
+              href="https://research.csc.fi/-/sd-connect"
+              :alt="$t('message.footerMenu.servicedescription')"
+            >{{
+              $t("message.footerMenu.servicedescription")
+            }}</a>
+          </div> 
+          <div class="navbar-item smalltext">
+            <p>{{ $t("message.footerMenu.accessibility") }}</p> 
+          </div>
+        </div>
+      </div>
+    </nav>
+  </div>
+    
+</template>
+
+<script>
+
+export default {
+  name:"CFooter",
+};
+</script>
+
+<style lang="scss" scoped>
+@import "@/css/prod.scss";
+
+
+#c-footer{
+  height:  fit-content;
+  width: 100%;
+  padding:40px 60px;
+  background-color:#DFE1E3;
+  z-index: 2;
+}
+
+.navbar{
+  background-color:#DFE1E3;
```

Comment:
I think since we also have `#DFE1E3` in here https://github.com/CSCfi/swift-browser-ui/blob/feature/new-footer/swift_browser_ui_frontend/src/css/csc.scss#L20, maybe we could use it as a variable instead ?

Reply:
Thank you @hannyle. I will fix those.